apps/devhub/tests/test_utils.py

```diff
@@ -0,0 +1,373 @@
+import os.path
+from copy import deepcopy
+
+import mock
+import pytest
+from mock import call
```

Comment:
I'd rather not have this import, and use `mock.call` later in the two places where it's used